websocket/src/websocket.cpp

```diff
@@ -436,6 +545,27 @@ void HandleCallback(WebsocketConnection* conn, int event, int msg_offset, int ms
         conn->m_HandshakeResponse = 0;
     }
 
+    dmSocket::Address address;
+    uint16_t port;
+    dmSocket::GetName(conn->m_Socket, &address, &port);
```

Comment:
This not worked in web build.
Also it worked on last week. But at 29.04.2024 i get errors.
INFO:WEBSOCKET: dmSocket::GetName address: 0x184f200 dmloader.js:524:37
ERROR:DLIB: Failed to retrieve address family (-4): BADF dmloader.js:525:40
ERROR:DLIB: Failed to retrieve address family (-4): BADF dmloader.js:525:40
ERROR:DLIB: Failed to retrieve socket information, unsupported address family! dmloader.js:525:40
INFO:WEBSOCKET: dmSocket::GetName address: 0x65f6c0, port: 0x65f6be